src/ng/directive/input.js

```diff
@@ -1436,6 +1481,23 @@ function numberInputType(scope, element, attr, ctrl, $sniffer, $browser) {
   }
 }
 
+function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
+  numberInputType(scope, element, attr, ctrl, $sniffer, $browser);
+
+  var originalRender = ctrl.$render;
+
+  ctrl.$render = function() {
+    if (ctrl.$isEmpty(ctrl.$viewValue)) {
+      ctrl.$viewValue = '50';
```

Comment:
BTW, i case it wasn't clear, `50` is the result of `(max - min) / 2` for the default min (`0`) and max (`100`) values.